tm2/pkg/std/memfile_test.go

```diff
@@ -8,9 +8,9 @@ import (
 
 func TestMemPackage_Validate(t *testing.T) {
 	tt := []struct {
-		name        string
-		mpkg        *MemPackage
-		errContains string
+		name           string
+		mpkg           *MemPackage
+		shouldHaveErr bool
```

Comment:
Why did you go from `errContains` to `shouldHaveErr`?

Note, I see manfred previously commented this suggesting to change from `errMayContains` to `shouldHaveErr`; but really, this original test already mentioned what's the better option: just use `errContains`, and check for `assert.ErrorContains`. This helps to visually illustrate _which_ error condition is hit.

Reply:
I have just updated the test :)
I think both `shouldHaveErr` and `errContains` is good for this case, so I kept it both.

Reply:
I also move the length limitation check toward to the top. I think this should match before checking for regex.